### PR TITLE
Adds initial copy / paste support.

### PR DESCRIPTION
**Pull request by diegoreymendez**
Related to #75.

Adds initial copy / paste support.  This PR is a bit large but at least half of the added lines are unit tests.

**Known limitations:**

When copy-pasting styles, there's some HTML node duplication (example: `<b>hello</b><b> there</b>`).  This'll be addressed in an upcoming PR, so please ignore for now.

**How to test:**

Run the unit tests... then:

1. Launch the example app.
2. Copy styled text.
3. Paste it into the editor.
4. Switch to HTML mode and make sure the tags are there (the supported ones are: bold, strike, underline, italic).

### CODE REVIEW

File: Aztec/Classes/Public/TextKit/TextStorage.swift [hunk 1]
```diff
@@ -111,7 +115,111 @@ public class TextStorage: NSTextStorage {
...
+        
+        guard let style = NSUnderlineStyle(rawValue: intValue) else {
+            assertionFailure("The strikethrough style value is not-known.")
+            return
+        }
+        
+        copyStrikethroughStyleToDOM(spanning: range, style: style)
+    }
+    
+    private func copyStrikethroughStyleToDOM(spanning range: NSRange, style: NSUnderlineStyle) {
```
Comment (SergioEstevao) [inline Aztec/Classes/Public/TextKit/TextStorage.swift]: Is this NSUnderlineStyle type for style correct here?

File: Aztec/Classes/Public/TextKit/TextStorage.swift [hunk 2]
```diff
@@ -111,7 +115,111 @@ public class TextStorage: NSTextStorage {
...
+        
+        switch (style) {
+        case .StyleSingle:
+            enableStrikethroughInDOM(range)
+        default:
+            // We don't support anything more than single-line underline for now
```
Comment (SergioEstevao) [inline Aztec/Classes/Public/TextKit/TextStorage.swift]: This comment looks wrong here.

File: Aztec/Classes/Public/TextKit/TextStorage.swift (hunk 1 above)
Comment (diegoreymendez) [inline Aztec/Classes/Public/TextKit/TextStorage.swift]: It seems counter-intuitive but it's indeed right.  From the documentation of `NSUnderlineStyle`:

> // This defines currently supported values for NSUnderlineStyleAttributeName and NSStrikethroughStyleAttributeName. NSUnderlineStyle*, NSUnderlinePattern*, and NSUnderlineByWord are or'ed together to produce an underline style.

File: Aztec/Classes/Public/TextKit/TextStorage.swift (hunk 2 above)
Comment (diegoreymendez) [inline Aztec/Classes/Public/TextKit/TextStorage.swift]: Good catch.

**Comment by diegoreymendez** [comment]
@SergioEstevao - Ready for round 2.

**Pull request merged**